### PR TITLE
fix(DB/Loot): Added loot to Training Dummy.

### LINKED ISSUE #10554: Target Dummies fail to drop loot
https://github.com/chromiecraft/chromiecraft/issues/2968

### What client do you play on?

enGB

### Faction

- [ ] Alliance
- [X] Horde

### Content Phase:

- [X] Generic
- [ ] 1-19
- [ ] 20-29
- [ ] 30-39
- [ ] 40-49
- [ ] 50-59
- [ ] 60

### Current Behaviour

Engineering Target Dummy (https://wowgaming.altervista.org/aowow/?item=4366) drops no loot.
They are marked as being lootable by specific party members, but are unable to be.
I've used like, three whole stacks of the things and not gotten a single drop from them, both in a party and not.

### Expected Blizzlike Behaviour

Target Dummy drops loot.
That's it.

### Source

_No response_

### Steps to reproduce the problem

1. Craft Target Dummy
2. Place Target Dummy
3. Fail to get drops when it dies.

### Extra Notes

_No response_

### AC rev. hash/commit

https://github.com/chromiecraft/azerothcore-wotlk/commit/6f20b9ece71984f1a11469e65ede8accda05653e

### Operating system

Ubuntu 20.04

Not sure if this means server or my personal OS...

### Modules

- [mod-ah-bot](https://github.com/azerothcore/mod-ah-bot)
- [mod-cfbg](https://github.com/azerothcore/mod-cfbg)
- [mod-chat-transmitter](https://github.com/azerothcore/mod-chat-transmitter)
- [mod-chromie-xp](https://github.com/azerothcore/mod-chromie-xp)
- [mod-desertion-warnings](https://github.com/azerothcore/mod-desertion-warnings)
- [mod-duel-reset](https://github.com/azerothcore/mod-duel-reset)
- [mod-eluna-lua-engine](https://github.com/azerothcore/mod-eluna-lua-engine)
- [mod-ip-tracker](https://github.com/azerothcore/mod-ip-tracker)
- [mod-low-level-arena](https://github.com/azerothcore/mod-low-level-arena)
- [mod-multi-client-check](https://github.com/azerothcore/mod-multi-client-check)
- [mod-pvp-titles](https://github.com/azerothcore/mod-pvp-titles)
- [mod-pvpstats-announcer](https://github.com/azerothcore/mod-pvpstats-announcer)
- [mod-queue-list-cache](https://github.com/azerothcore/mod-queue-list-cache)
- [mod-server-auto-shutdown](https://github.com/azerothcore/mod-server-auto-shutdown)
- [mod-save-inventory](https://github.com/Nyeriah/mod-save-inventory)
- [lua-carbon-copy](https://github.com/55Honey/Acore_CarbonCopy)
- [lua-exchange-npc](https://github.com/55Honey/Acore_ExchangeNpc)
- [lua-custom-worldboss](https://github.com/55Honey/Acore_CustomWorldboss)
- [lua-level-up-reward](https://github.com/55Honey/Acore_LevelUpReward)
- [lua-recruit-a-friend](https://github.com/55Honey/Acore_RecruitAFriend)
- [lua-send-and-bind](https://github.com/55Honey/Acore_SendAndBind)
- [lua-temp-announcements](https://github.com/55Honey/Acore_TempAnnouncements)
- [lua-zonecheck](https://github.com/55Honey/acore_Zonecheck)
- [lua-zone-debuff](https://github.com/55Honey/Acore_ZoneDebuff)

### Customizations

None

### Server

ChromieCraft

### PR DESCRIPTION
**Pull request by UltraNix**
Fixes #10554

## Issues Addressed:

- Closes #10554
- Closes https://github.com/chromiecraft/chromiecraft/issues/2968

## Tests Performed:

- Not tested.

## How to Test the Changes:

Craft Target Dummy
Place Target Dummy
Kill dummy and see the loot.

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

File: data/sql/updates/pending_db_world/rev_1644689254263193400.sql [hunk 1]
```diff
@@ -0,0 +1,11 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1644689254263193400');
+
+UPDATE `creature_template` SET `lootid`=2673 WHERE `entry`=2673;
+
+DELETE FROM `creature_loot_template` WHERE `entry`=2673;
+INSERT INTO `creature_loot_template` VALUES
+(2673,2592,0,0,0,1,1,1,1,'Training Dummy'),
```
[the remark this answers is not in the document]
Comment (acidmanifesto) [reply data/sql/updates/pending_db_world/rev_1644689254263193400.sql:7]: im sorry. i forgot this is dealing with loot. i didnt mean reference as in reference table, but a source dealing with what lead to the decision of the loot template here.

[the remark this answers is not in the document]
Comment (UltraNix) [reply data/sql/updates/pending_db_world/rev_1644689254263193400.sql:7]: The issue that is linked to that PR? I don't understand your question.

[the remark this answers is not in the document]
Comment (grimgravy) [reply data/sql/updates/pending_db_world/rev_1644689254263193400.sql:7]: it must be this source [WowHead Loot](https://classic.wowhead.com/npc=2673/target-dummy)

**Comment by IntelligentQuantum** [comment]
@acidmanifesto Merge OR ?

**Comment by acidmanifesto** [comment]
> @acidmanifesto Merge OR ?

given there is no sniffs but a few sites referencing a loot we can merge it. I am just not entirely certain where the % is coming from.

**Comment by grimgravy** [comment]
> > @acidmanifesto Merge OR ?
> 
> given there is no sniffs but a few sites referencing a loot we can merge it. I am just not entirely certain where the % is coming from.

Bronze [chance: 64% drop](https://classic.wowhead.com/item=2841/bronze-bar#dropped-by)
Handful Ofcopper Bolts [chance: 79% drop](https://classic.wowhead.com/item=4359/handful-of-copper-bolts)
Copper Modulator [chance: 36% drop](https://classic.wowhead.com/item=4363/copper-modulator)
Linen Cloth [chance: 71% drop](https://classic.wowhead.com/item=2589/linen-cloth)

**Review by acidmanifesto — APPROVED** [review]

**Pull request merged**